# Zend Framework 1 - Exception

[![Build](https://github.com/diablomedia/zf1-exception/workflows/Build/badge.svg?event=push)](https://github.com/diablomedia/zf1-exception/actions?query=workflow%3ABuild+event%3Apush)
[![codecov](https://codecov.io/gh/diablomedia/zf1-exception/branch/master/graph/badge.svg)](https://codecov.io/gh/diablomedia/zf1-exception)
[![Latest Stable Version](https://poser.pugx.org/fragotesac/zf1-exception/v/stable)](https://packagist.org/packages/fragotesac/zf1-exception)
[![Total Downloads](https://poser.pugx.org/fragotesac/zf1-exception/downloads)](https://packagist.org/packages/fragotesac/zf1-exception)
[![License](https://poser.pugx.org/fragotesac/zf1-exception/license)](https://packagist.org/packages/fragotesac/zf1-exception)

This is just the Zend_Exception component extracted from our fork of the Zend Framework 1 repo (https://github.com/diablomedia/zf1).
